rero_ils/modules/utils.py

```diff
@@ -303,7 +314,7 @@ def generate_item_barcode(data=None):
         data['barcode'] = 'f-{}'.format(
             datetime.now().strftime('%Y%m%d%I%M%S%f'))
     if data.get('patron') and not data.get('patron', {}).get('barcode'):
-        data['patron']['barcode'] = 'f-{}'.format(
+        data['patron']['barcode'][0] = 'f-{}'.format(
```

Comment:
data['patron']['barcode'] = ['f-{}'.format(
            datetime.now().strftime('%Y%m%d%I%M%S%f'))]

Reply:
You are right. But as the barcode is required for a patron, this case should never happens! Can I remove this?